src/assets/components/Form/Form.tsx

```diff
@@ -0,0 +1,87 @@
+import "./Form.css";
+import React, { useState } from "react";
+
+interface Props {
+  onSubmit: (data: {
+    id: string;
+    name: string;
+    difficulty: string;
+    price: number;
+  }) => void;
+}
+
+export const Form: React.FC<Props> = ({ onSubmit }) => {
+  const [name, setName] = useState("");
+  const [difficulty, setDifficulty] = useState("");
+  const [price, setPrice] = useState<number | "">("");
```

Comment:
why this type looks like this `<number | "">` ? It could stay so, just for clarity

Reply:
It's because I want to put placeholder on this input and for this I use this syntax(number |(or) "")